netdata.spec.in

```diff
@@ -129,27 +129,29 @@ fi
 rm -rf $RPM_BUILD_ROOT
 
 %files
-%attr(-,netdata,netdata) %dir %{_localstatedir}/cache/%{name}
-%attr(-,netdata,netdata) %dir %{_localstatedir}/log/%{name}
+%defattr(-,root,root)
+
+%dir %{_sysconfdir}/%{name}
 %config(noreplace) %{_sysconfdir}/%{name}/*.conf
 %config(noreplace) %{_sysconfdir}/%{name}/python.d/*.conf
-%dir %{_sysconfdir}/%{name}
-%{_sysconfdir}/logrotate.d/%{name}
+%config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
+
 %{_libexecdir}/%{name}
 %{_sbindir}/%{name}
+
+%attr(0700,netdata,netdata) %dir %{_localstatedir}/cache/%{name}
+%attr(0700,netdata,netdata) %dir %{_localstatedir}/log/%{name}
+%attr(0700,netdata,netdata) %dir %{_localstatedir}/lib/%{name}
+
 %dir %{_datadir}/%{name}
-%dir %{_sharedstatedir}/%{name}
+%attr(-,root,netdata) %{_datadir}/%{name}/web
```

Comment:
this is not exactly the same as playing with `%defattr(644,root,netdata,755)`

with defattr one can force 644 for files and 755 for dirs. what you do is to assume permission bits what `make install` produced. this is probably fine, but double check that the files under web are permissions intended to.